rdrf/rdrf/initial_data/blacklisted_mime_types.py

```diff
@@ -0,0 +1,19 @@
+"""
+Blacklisted mime types setup
+"""
+
+from rdrf.models.definition.models import BlacklistedMimeType
+
+
+def load_data(**kwargs):
+    init_blacklisted_file_types()
+
+
+def init_blacklisted_file_types():
+
+    BlacklistedMimeType.objects.get_or_create(
+        mime_type='application/octet-stream',
```

Comment:
Can you please do a bit of research about what other files are considered "dangerous"?
To test I started with going to ``/bin/`` uploading some executables. ``application/x-executable``, and ``text/x-shellscript`` were accepted, when they shouldn't and it raises the question what other mime-types we shouldn't allow for sure.
Worth investigating a bit...

Reply:
Added more mime-types to the initial data script to cover more file types.